docs/api/app.md

```diff
@@ -927,6 +927,27 @@ Enables mixed sandbox mode on the app.
 
 This method can only be called before app is ready.
 
+### `app.isInApplicationsFolder()` _macOS_
+
+Returns `Boolean` - Whether the application is currently running from the
+systems Application folder.  Use in combination with `app.moveToApplicationsFolder()`
+
+### `app.moveToApplicationsFolder()` _macOS_
+
+Returns `Boolean` - Whether the move was successful.  Please note that if
+the move was successful your application will be quit and relaunched
+automatically in it's new location.
+
+This method does not have any dialog or confirmation by default, if you want
+to ask your user if you should make this move then you should use the `dialog`
+API's.
```

Comment:
-> No confirmation dialog will be presented by default, if you wish to allow the user to confirm the operation you may do so using the `dialog` API.